pkg/network/network.go

```diff
@@ -245,40 +111,91 @@ func CreateTap(name string, mtu int, ownerUID, ownerGID int) (netlink.Link, erro
 		return nil, fmt.Errorf("failed to set tap device MTU to %d: %w", mtu, err)
 	}
 
-	err = netlink.LinkSetUp(tapLink)
+	return tapLink, nil
+}
+
+// ensureEth0Exists checks all network interface in current netns and returns
```

Comment:
A small type. "interface" -> "interfaces"

Reply:
Fixed it.